angular-hub/src/app/pages/index.page.ts

```diff
@@ -25,6 +25,14 @@ export const routeMeta: RouteMeta = {
       name: 'description',
       content: 'Curated list of Angular Events',
     },
+    {
+      property: 'og:title',
+      content: 'Title of the page',
```

Comment:
Um, this should be replaced I guess? 😅

Reply:
Haha @ajitzero good catch. I was too excited to test it out when missed that detail!